src/main/java/org/semux/api/v2/SemuxApiImpl.java

```diff
@@ -167,7 +168,9 @@ public Response composeRawTransaction(String network, String type, String to, St
                     .withData(data)
                     .withGas(gas)
                     .withGasPrice(gasPrice);
-            Transaction transaction = transactionBuilder.buildUnsigned();
+                       
+            /*We can use TO address instead FROM address, only if transaction will be used for ComposeRawTransactionResponse*/
```

Comment:
Can you explain this? i dont think using the to address works, as it'd result in a different hash.